tests/ImageSharp.Tests/Processing/Processors/Transforms/RotateTests.cs

```diff
@@ -39,6 +61,21 @@ public void Rotate<TPixel>(TestImageProvider<TPixel> provider, float value)
             }
         }
 
+        [Theory]
+        [WithTestPatternImages(nameof(ResmplerNames), 100, 50, DefaultPixelType)]
+        [WithTestPatternImages(nameof(ResmplerNames), 50, 100, DefaultPixelType)]
+        public void RotateWithSampler<TPixel>(TestImageProvider<TPixel> provider, string resamplerName)
+            where TPixel : struct, IPixel<TPixel>
+        {
+            IResampler resampler = GetResampler(resamplerName);
+
+            using (Image<TPixel> image = provider.GetImage())
+            {
+                image.Mutate(x => x.Rotate(50, resampler));
+                image.DebugSave(provider, resamplerName);
```

Comment:
If we are fine with the current results, it would be wise, to push the results into the reference image repository, and add `image.CompareToReferenceOutput(provider, ImageComparer.Tolerant());` here. At least for a few resamplers.

Reply:
I think you've found an issue. I'm not sampling the offset correctly .

I'm starting at zero when I should be starting at -radius. I'll have a look.

Reply:
Much better results now for the larger window samplers. We still get slight offsetting for those samplers but that's simply how they work and our output images are tiny so we notice it more. 

No-one in their right mind would use a Lanczos8 sampler for rotation anyway 😛.